content/articles/2019-02-dplyr-0-8-0.Rmarkdown

```diff
@@ -0,0 +1,79 @@
+---
+title: dplyr 0.8.0
+author: Romain François
+date: '2019-02-15'
+slug: dplyr-0-8-0
+categories:
+  - package
+tags:
+  - dplyr
+  - tidyverse
+  - package
+---
+
+```{r setup, include = FALSE}
+library(dplyr)
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+```
+
+We're tickled pink to announce the release of version 0.8.0 of [dplyr](https://dplyr.tidyverse.org), the grammar of data manipulation in the tidyverse. 
+This is a major update that has kept us busy for almost a year. We take the coincidence of a Valentine's day release as a sign 
+of continuous `r emo::ji("heart")` for `dplyr`'s approach to tidy data manipulation. 
+
+Important changes are discussed in detail in the [pre-release post](https://www.tidyverse.org/articles/2018/12/dplyr-0-8-0-release-candidate/), 
+we are grateful to members of the community for their feedback in the last couple of months, this 
+has been tremendously useful to make the release process seemless. 
+
+The bulk of the changes are internal, and part of an ongoing effort to make the codebase more robust and 
+less surprising. This is an investment that will continue to pay off for years, and serves as a foundation for 
+more innovations in the future. 
+
+For a comprehensive list of changes, please see [the NEWS section](https://dplyr.tidyverse.org/news/index.html#dplyr-0-8-0) 
+for the 0.8.0 release, the sections below discusses the main changes. 
+
+## Group hug
+
+Grouping has always been at the center of what `dplyr` is about, this release expands on the 
```

Comment:
Do we want to mention that these are still experimental, so we appreciate feedback?

Reply:
I like that idea!

Reply:
Went with 

```
We believe they offer new unique possibilities, but we welcome community feedback and use cases before we put a `r emo::ji("ring")` on them.
```

Should we encourage users to go on a few dates with the functions and get to know them ?